frontend/talentsearch/src/js/components/aboutMeForm/AboutMeForm.tsx

```diff
@@ -0,0 +1,352 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { toast } from "react-toastify";
+import { OperationResult } from "urql";
+import { BasicForm, Input, RadioGroup, Select } from "@common/components/form";
+import { ProvinceOrTerritory, Language } from "@common/api/generated";
+import { commonMessages, errorMessages } from "@common/messages";
+import { phoneNumberRegex } from "@common/constants/regularExpressions";
+import { enumToOptions } from "@common/helpers/formUtils";
+import { getLocale } from "@common/helpers/localize";
+import { navigate } from "@common/helpers/router";
+import {
+  getProvinceOrTerritory,
+  getLanguage,
+} from "@common/constants/localizedConstants";
+import { SubmitHandler } from "react-hook-form";
+import { omit, pick } from "lodash";
```

Comment:
One small suggestion, we've always tried to import from lodash submodules as its an easy way to slightly limit the size of the import.
```suggestion
import omit from "lodash/omit";
import pick from "lodash/pick";
```

Reply:
Thanks for the heads up, I'll make sure to do this in all future imports.